examples/Retrieval-Models.ipynb

```diff
@@ -0,0 +1,1528 @@
+{
```

Comment:
<p>I'd rather not create the BatchedDataset ourselves - and just pass the merlin.io.Dataset object to model.fit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
yes will do that.